src/components/Hero/Hero.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import "./Hero.scss";
+import { BrowserRouter as Router, Link } from "react-router-dom";
+import { Button } from "react-bootstrap";
+//Button is not working from react-bootstrap
```

Comment:
I think react css needs to be improted in the app.js in order to be able to use react-bootstrap

Reply:
@busesisi can we fix this ?